chromium_src/chrome/browser/ui/views/tabs/tab.cc

```diff
@@ -1,26 +0,0 @@
-/* Copyright (c) 2019 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "brave/browser/ui/views/tabs/brave_alert_indicator_button.h"
-
-// Set alert indicator's pos to start of the title and
-// move title after the alert indicator.
-// Title right should respect close btn's space
-#define BRAVE_UI_VIEWS_TABS_TAB_ALERT_INDICATOR_POSITION           \
```

Comment:
Oh, wait. I think this is not related with this PR?

Reply:
This change came from https://github.com/brave/brave-core/pull/9445 and it's for re-positioning alert button

Reply:
I see, I'll fix that. Thanks for catching!

Reply:
Fixed.